pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -663,11 +663,9 @@ class Build {
 
         context.node('worker') {
             context.stage("installer") {
-                // Ensure master context workspace is clean of any previous archives
-                context.sh "rm -rf workspace/target/* || true"
 
                 switch (buildConfig.TARGET_OS) {
-                    case "mac":     buildMacInstaller(versionData); break
+                    case "mac": context.sh 'rm -f workspace/target/*.pkg workspace/target/*.pkg.json workspace/target/*.pkg.sha256.txt'; buildMacInstaller(versionData); break
```

Comment:
In this case for windows previous archives will not be cleaned, which will be not clean the any previous workspace artifacts as PR https://github.com/adoptium/ci-jenkins-pipelines/pull/360/files expected. `Ensure node context workspace is clean prior to gpg signing `

Reply:
it's actually updated by https://github.com/adoptium/ci-jenkins-pipelines/pull/340/files

Reply:
yeah, the reason first  to add `rm -rf workspace/target/*` is because the code was running on the "master|built-in" (multiple nodes, than before was only one master)
then later it changed from "master|built-in" to "node" which should not have the problem.